root/static/scripts/edit/externalLinks.js

```diff
@@ -196,6 +196,11 @@ class ExternalLinksEditor extends React.Component<LinksEditorProps, LinksEditorS
               error = l('This relationship type is deprecated and should not be used.');
             } else if ((!isPositiveInteger(link.relationship) || (oldLink && link.url !== oldLink.url)) && checker && !checker(link.url)) {
               error = l('This URL is not allowed for the selected link type, or is incorrectly formatted.');
+            } else if ((!isPositiveInteger(link.relationship) || (oldLink && link.url !== oldLink.url)) && /^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\/.*#/.test(link.url)) {
+              // Kludge for MBS-9515 to be replaced with the more general MBS-9516
+              error = l('Anchor (#Example_section) Wikipedia links are not allowed, see the {url|guidelines for URL}.', {
+                url: { href: '/doc/Style/Relationships/URLs', target: '_blank' }
```

Comment:
This should probably link to something like https://musicbrainz.org/relationship/29651736-fa6d-48e4-aadc-a557c6add1cb 

It's kind of annoying that we don't have a central "Wikipedia page" rel to link to, but only one per entity type :/ But linking to a specific one, like artist, might be better than nothing.

Reply:
https://github.com/metabrainz/musicbrainz-server/pull/562/commits/46feadc2646625daf81abd03ce12adc529f33f23 links to the specific relationship page for the current URL.
Besides, I reported the lack of central page for similar relationship types in [MBS-9517](https://tickets.metabrainz.org/browse/MBS-9517).